### PR TITLE
Add README.md

### PR DESCRIPTION
**Pull request by aliuk2012**
This is a very basic README file that will be fleshed out further as we start to use the npm module.

### Still to added:
- Install instructions
- Local development process (building custom components)
- Coding standards

Ticket: https://trello.com/c/KBZFQOfK

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+This repository does two things:
+
+- provides a central repository for custom components used in Digital Marketplace.
+- Imports GOV.UK Frontend and serves it to Digital Marketplace Frontend applications
+  where they can consume GOV.UK Frontend and all the custom components in a single dependency
+
+The problem we are trying to solve:
+
+- its difficult to ensure that all our applications are using the same version of GOV.UK Frontend and digitalmarketplace-frontend-toolkit (which is being replaced).
```
Comment (lfdebrux) [inline README.md]:
```suggestion
- it is difficult to ensure that all our applications are using the same version of GOV.UK Frontend and digitalmarketplace-frontend-toolkit (which is being replaced).
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+- its difficult to ensure that we are only importing and using css/js for components we are actually using and not been removed.
```
Comment (lfdebrux) [inline README.md]:
```suggestion
- it is difficult to ensure that we are only importing and using css/js for components we are actually using and not been removed.
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+GOV.UK Frontend is maintained by a team at Government Digital Service. If you want to know more about GOV.UK Frontend, please email the [Design System
+team](mailto:govuk-design-system-support@digital.cabinet-office.gov.uk) or get in touch with them on [Slack](https://ukgovernmentdigital.slack.com/messages/govuk-design-system).
+
+## Digital Marketplace GOV.UK Frontend
+
+Digital Marketplace GOV.UK Frontend contains the code you need to start building a user interface for Digital Marketplace.
+
+### Contact the Digital Marketplace team
+
+Digital Marketplace GOV.UK Frontend is maintained by a team at Government Digital Service. If you want to know more about Digital Marketplace GOV.UK Frontend, please email the [Digital Marketplace team](mailto:digital-marketplace-development@digital.cabinet-office.gov.uk)
```
Comment (lfdebrux) [inline README.md]:
```suggestion
Digital Marketplace GOV.UK Frontend is maintained by a team at Government Digital Service. If you want to know more about Digital Marketplace GOV.UK Frontend, please email the [Digital Marketplace team](mailto:digital-marketplace-development@digital.cabinet-office.gov.uk).
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
+Digital Marketplace GOV.UK Frontend ·
+[![JavaScript Style Guide](https://img.shields.io/badge/code_style-standard-brightgreen.svg)](https://standardjs.com)
+[![Known Vulnerabilities](https://snyk.io//test/github/alphagov/digitalmarketplace-govuk-frontend/badge.svg?targetFile=package.json)](https://snyk.io//test/github/alphagov/digitalmarketplace-govuk-frontend?targetFile=package.json)
+=====================
+
+This repository does two things:
+
+- provides a central repository for custom components used in Digital Marketplace.
```
Comment (lfdebrux) [inline README.md]:
```suggestion
- it provides a central repository for custom components used in the Digital Marketplace
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+- Imports GOV.UK Frontend and serves it to Digital Marketplace Frontend applications
```
Comment (lfdebrux) [inline README.md]:
```suggestion
- it imports GOV.UK Frontend and serves it to Digital Marketplace frontend applications
```

File: README.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
+Digital Marketplace GOV.UK Frontend ·
+[![JavaScript Style Guide](https://img.shields.io/badge/code_style-standard-brightgreen.svg)](https://standardjs.com)
+[![Known Vulnerabilities](https://snyk.io//test/github/alphagov/digitalmarketplace-govuk-frontend/badge.svg?targetFile=package.json)](https://snyk.io//test/github/alphagov/digitalmarketplace-govuk-frontend?targetFile=package.json)
```
Comment (katstevens) [inline README.md]: I think the double-slash in the URL is causing a broken image:

readme-header

File: README.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+GOV.UK Frontend is maintained by a team at Government Digital Service. If you want to know more about GOV.UK Frontend, please email the [Design System
+team](mailto:govuk-design-system-support@digital.cabinet-office.gov.uk) or get in touch with them on [Slack](https://ukgovernmentdigital.slack.com/messages/govuk-design-system).
+
+## Digital Marketplace GOV.UK Frontend
+
+Digital Marketplace GOV.UK Frontend contains the code you need to start building a user interface for Digital Marketplace.
+
+### Contact the Digital Marketplace team
+
+Digital Marketplace GOV.UK Frontend is maintained by a team at Government Digital Service. If you want to know more about Digital Marketplace GOV.UK Frontend, please email the [Digital Marketplace team](mailto:digital-marketplace-development@digital.cabinet-office.gov.uk).
```
Comment (katstevens) [inline README.md]: I don't know if we put contact details on any of our other repos! Is this a trend we want to start? 😹

File: README.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+- [watch the digitalmarketplace-govuk-frontend Github repository](https://help.github.com/en/articles/watching-and-unwatching-repositories)
+
+Find out how to [update with npm](docs/installation/updating-with-npm.md).
+
+## Licence
+
+Unless stated otherwise, the codebase is released under the MIT License. This
+covers both the codebase and any sample code in the documentation. The
+documentation is &copy; Crown copyright and available under the terms of the
+Open Government 3.0 licence.
```
Comment (katstevens) [inline README.md]: We will need to add a separate license file for the repo anyway, so we don't need this paragraph. See https://github.com/alphagov/digitalmarketplace-utils/blob/master/LICENCE for an example. I'm happy to do this in a separate PR.

File: docs/installation/updating-with-npm.md [hunk 9]
```diff
@@ -0,0 +1,38 @@
+# Updating Digital Marketplace GOV.UK Frontend using Node.js package manager (npm)
+
+You can update with Node.js package manager (npm) if you [originally installed
```
Comment (katstevens) [inline docs/installation/updating-with-npm.md]: `You can update...` Update what?
